parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -654,9 +654,9 @@ notifyUser dir o = case o of
       where
       push = P.group . P.backticked $ IP.patternName IP.push
       pull = case input of
-        Input.PushRemoteBranchI Nothing p ->
+        Input.PushRemoteBranchI Nothing p _ ->
```

Comment:
Is this correct? Should you be switching off the mode here to produce different output?

Reply:
Oops, good catch.